### ISSUE TITLE
[TorrentCCF] Frequent need to resolve Captcha Image

### ISSUE THREAD

**Issue by RoloSoze** [Jackett/Jackett#6876]
With TorrentCCF, I need to reconfigure the tracker every few days by resolving the Captcha Image. Would the cookie method solve this?

Jackett version:  0.12.1494.0
Mono version: N/A

**Comment by garfield69** [comment]
unfortunately not.
# TLDR;
cookies from a site have expiry date/times, and if the site determines that a cookie is too old it will request a re-login.

# FYI
when you interact with a web site, the site may ask your browser to store cookies. these a keywords and values that the site want the browser to store, so that the next time you visit the site it can use them for what ever purpose.
these cookies are set by the site with an expiry date/time.
after they expire the site will issue new cookies.

when you login with your browser using a  login form with a user/password, the site will store some cookies containing your session-id and what ever other cookie keywords the site uses to identify you on your next visit (be that simply you browsing the sites torrent search page, or what ever).

the jackett cookie login method is a way to bypass the login page when the site is using a login form that jackett cannot use for what ever reason (reCaptha challenge in this case).
so instead of using the login form, jackett just goes straight to the torrent search page and passes any cookies that you stored in the jackett config for that indexer.
And as long as the cookies have not expired the site will accept them and you are good to go.
If however the site determines that the cookies have expired, then it will request a re-login.

and that is why the jackett cookie login is not a magic bullet.

**Comment by garfield69** [comment]
Jackett 0.16.515

**Comment by RoloSoze** [comment]
> 
> 
> Jackett 0.16.515

Installed, reconfigured both the PTer and TorrentCCF indexers. The problem is resolved for PTer but the issue remains the same for TorrentCCF. I still have to reconfigure upon IP change.

**Comment by garfield69** [comment]
Oops, somehow I did not save my torrentccf update. I'll try again.

**Comment by garfield69** [comment]
Jackett 0.16.521
